### DISCUSSION TITLE
Screen gets frozen while loading

### DISCUSSION THREAD

**Discussion by maldeek** [ludwig-v/wireless-carplay-dongle-reverse-engineering#152] [General]
Hello awesome people,

I have a big issue that happens frequently, whenever my iPhone is connected to the Carlinkit model QL-CP2A-LOGO, the screen gets frozen on the page connecting or sometimes it manages to connect, but then Carplay is not responding to the touch. I tried connecting directly via cable without the dongle and it works fine. 

By the way, the car is Seat Leon with OEM screen.

Would really love to get this sorted.

On another note, this issue happens even more with Android phone, whenever phone is connected, it requires dozen tries until I get the chance to see the Android Auto.

**Comment by michalmcz** [comment]
Hello,

I'm facing same issue with my Carplay. 

Sometimes it connects without any issue, sometime not and it needs to be restart/unplug.

Any idea?

Thanks a lot,
M.

**Comment by domschmidt** [comment]
Tiguan 2019 with Discovery Pro / Iphone 13 Pro Max
Screen Freezes & Bluetooth disconnect & no touchscreen input possible

In the facebook discussion they offer a new version for V4 (2023.03.10.0903) that solves my problem:
https://www.facebook.com/groups/carlinkitservices/permalink/1591862631316708

**Reply by ludwig-v** [reply]
Thank's for the report, I added the firmware on the repo ;)
